ldk/go/examples/network/readme.md

```diff
@@ -0,0 +1,59 @@
+# Sidekick Loop Example in Go
+
+This is a very simple loop to use as a starting point. This loop watches for text events and emits a whisper every time it receives an event.
```

Comment:
Is this true or does it send a network request?

Reply:
This was copied from another example... So 🤷‍♂️ I'll look into updating this.